vru_use_case_scripts/plot_sdsm_position_error

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python3
+
+from argparse import ArgumentParser
+from collections import defaultdict
+import json
+from pathlib import Path
+import sys
+
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+
+
+def parse_positions_from_sdsm_csv(file_path: Path):
+    def get_timestamp(line: str):
+        return round(float(line.split(",")[1]))
+
+    def get_json_payload(line: str):
+        json_start = line.find(',"') + 2
+        json_end = line.rfind('"')
+
+        # JSON string needs double quotes
+        return line[json_start:json_end].replace("'", '"')
+
+    sdsm_data = []
+    with open(file_path) as file:
```

Comment:
I don't think `DataFrame`s would be appropriate as an intermediate structure here because we need to line-level parsing to extract the detected objects from each SDSM entry.

We would have to create an initial `DataFrame` from the CSV data then iterate through it, adding new lines for each detected object and removing the original line afterwards.

Reply:
I replace the manual string splitting with the `csv` package, which makes parsing a little more robust.